client/my-sites/checkout/composite-checkout/components/item-variation-picker/variant-dropdown-price.tsx

```diff
@@ -74,12 +75,18 @@ const IntroPricing = styled.span`
 	display: flex;
 	flex-direction: column;
 	font-size: 0.8rem;
-	margin-left: auto;
-	p {
-		text-align: right;
+	.div {
+		display: block;
+		margin-bottom: 0rem;
 	}
```

Comment:
These lines seem like they might be redundant, now that `IntroPricingText` is a thing starting on line 84.